### PR TITLE
LPS-163661 | Test Automation | Master

### PR DESCRIPTION
**Pull request by marcosvictor22**
[LPS-163661](https://issues.liferay.com/browse/LPS-163661)

### CODE REVIEW

**Comment by marcosvictor22** [comment]
ci:test:sf

**Comment by JhossephG** [comment]
Good work on this automation @marcosvictor22! You just need change you task blocks to use the BDD structure [example](https://github.com/liferay-bpm-qa/liferay-portal/pull/282/files) and please add the Index.html at the ticket.

**Comment by epark18** [comment]
@marcosvictor22 Good job on this. Like @JhossephG mentioned it needs to follow BDD format. So something like 

1. Given an object with self relationship and cascade deletion type
2. And given a default layout is created to display relationships
3. And given multiple entries are created and have a hierarchical relationship
4. When the user deletes a parent entry
5. Then the entry and it's child are deleted.

Also since this a priority 5, the test needs to be added to acceptance with the property _property portal.acceptance = "true";_

**Comment by marcosvictor22** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase [hunk 1]
```diff
@@ -390,6 +390,120 @@ definition {
...
+			CreateObject.addRelationshipAndSelectEntry(titleFieldName = "Entry B");
+
+			ObjectPortlet.viewEntryDetails(entry = "Entry B");
+
+			ObjectAdmin.gotoRelationshipsTab();
+
+			CreateObject.addRelationshipAndSelectEntry(titleFieldName = "Entry C");
+		}
+
+		task ("And when the user deletes a parenty entry") {
```
Comment (JhossephG) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase:488]: Just fix the typo on this task :)

**Comment by JhossephG** [comment]
Good job @marcosvictor22! When you fix the typo that I mentioned and @epark18 give the green flag you can forward🎉

**Comment by marcosvictor22** [comment]
ci:test:sf

**Comment by epark18** [comment]
ci:forward

**Pull request closed without merging**